docs/ref/django-admin.txt

```diff
@@ -647,6 +647,21 @@ Suppresses writing '``#: filename:line``’ comment lines in language files.
 Using this option makes it harder for technically skilled translators to
 understand each message's context.
 
+.. django-admin-option:: --add-location [{full,file,never}]
+
+Controls '``#: filename:line``’ comment lines in language files.
+
+The optional type can be either ``full``, ``file``, or ``never``. If it is not
+given or ``full``, it generates the lines with both file name and line number.
+If it is ``file``, the line number part is omitted. If it is ``never``, it
+completely suppresses the lines (same as ``--no-location``).
+
+Example usage::
+
+    django-admin makemessages --add-location
+    django-admin makemessages --add-location file
+    django-admin makemessages --add-location=never
+
```

Comment:
Added in e3d0214. Please see: https://github.com/django/django/pull/8333/commits/e3d021459e90e5be92c86c6366e645ccb3f60bd8#diff-b5ea25a58e9c8adf1a328182844020caR652